lib/msf/core/post/windows/priv.rb

```diff
@@ -50,15 +50,12 @@ def initialize(info = {})
   def is_admin?
     if session_has_ext
       # Assume true if the OS doesn't expose this (Windows 2000)
-      session.railgun.shell32.IsUserAnAdmin()["return"] rescue true
-    else
-      local_service_key = registry_enumkeys('HKU\S-1-5-19')
-      if local_service_key
-        return true
-      else
-        return false
-      end
+      return session.railgun.shell32.IsUserAnAdmin()['return'] rescue true
     end
+
+    local_service_key = registry_enumkeys('HKU\S-1-5-19')
+
+    !local_service_key.blank?
```

Comment:
```suggestion
    !local_service_key.empty?
```

Array's in Ruby don't seem to have a `.blank?` method but they do have a `.empty?` method which is what I think you were after here:

```
 ~/git/metasploit-framework │ master ?27  irb                     ✔ │ 1m 26s │ 3.0.2 Ruby 
3.0.2 :001 > [].blank?
(irb):1:in `<main>': undefined method `blank?' for []:Array (NoMethodError)
	from /home/gwillcox/.rvm/gems/ruby-3.0.2@metasploit-framework/gems/irb-1.3.6/exe/irb:11:in `<top (required)>'
	from /home/gwillcox/.rvm/gems/ruby-3.0.2@metasploit-framework/bin/irb:23:in `load'
	from /home/gwillcox/.rvm/gems/ruby-3.0.2@metasploit-framework/bin/irb:23:in `<main>'
	from /usr/share/rvm/rubies/ruby-3.0.2/bin/ruby_executable_hooks:22:in `eval'
	from /usr/share/rvm/rubies/ruby-3.0.2/bin/ruby_executable_hooks:22:in `<main>'
3.0.2 :002 > [].empty?
 => true 
3.0.2 :003 > 
```

Reply:
Nm this is a Rails method which will be accessible from within Metasploit. Ignore my comments here.